posthog/session_recordings/queries/session_recording_list_from_replay_summary.py

```diff
@@ -1,785 +0,0 @@
-import dataclasses
-import re
-from datetime import datetime, timedelta
-from typing import Any, Literal, NamedTuple, Union
-
-from django.conf import settings
-from sentry_sdk import capture_exception
-
-from posthog.client import sync_execute
-from posthog.constants import TREND_FILTER_TYPE_ACTIONS, PropertyOperatorType
-from posthog.models import Entity, Team
-from posthog.models.action.util import format_entity_filter
```

Comment:
Looks like we were the only ones using this in the codebase. Should probably remove it too